client/components/add-new-site/menu-items/site-list.tsx

```diff
@@ -0,0 +1,139 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import config from '@automattic/calypso-config';
+import page from '@automattic/calypso-router';
+import { WordPressLogo, JetpackLogo } from '@automattic/components';
+import { download, reusableBlock, Icon } from '@wordpress/icons';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import { A4A_MARKETPLACE_HOSTING_PRESSABLE_LINK } from 'calypso/a8c-for-agencies/components/sidebar-menu/lib/constants';
+import useFetchDevLicenses from 'calypso/a8c-for-agencies/data/purchases/use-fetch-dev-licenses';
+import usePressableOwnershipType from 'calypso/a8c-for-agencies/sections/marketplace/hosting-overview/hooks/use-pressable-ownership-type';
+// TODO: This will need to be updated to use whatever image we decide on.
+import devSiteBanner from 'calypso/assets/images/a8c-for-agencies/dev-site-banner.svg';
+import pressableIcon from 'calypso/assets/images/pressable/pressable-icon.svg';
+import AddNewSiteMenuItem from 'calypso/components/add-new-site/menu-item';
+import AddNewSitePopoverColumn from 'calypso/components/add-new-site/popover-column';
+import { preventWidows } from 'calypso/lib/formatting';
+import { TRACK_SOURCE_NAME } from 'calypso/sites-dashboard/utils';
+
+const AddNewSiteSiteListMenuItems = () => {
+	const translate = useTranslate();
+
+	const pressableOwnership = usePressableOwnershipType();
+
+	const { data: devLicenses } = useFetchDevLicenses();
+
+	const hasAvailableDevSites = devLicenses?.available > 0;
+
+	const devSitesEnabled = config.isEnabled( 'a4a-dev-sites' );
```

Comment:
```suggestion
	const devSitesEnabled = config.isEnabled( 'sites/drive-migrations' );
```

Reply:
I don't think this should be tied to the config. I think it should be it's own check in case there's other criteria we want to use for displaying it.

How about I just set it to `true` for now?

Reply:
can we remove the whole devSites logic? Or that would serve some purpouse on DotCom?

Reply:
Sorry, I didn't explain it very well. I changed it from `devSitesEnabled` (and `hasAvailableDevSites`) to `popoverCardEnabled` and `popoverOfferEnabled` and defaulted both of them to be true. That way it shows the intent better.